planning_scene/src/planning_scene.cpp

```diff
@@ -1133,8 +1133,10 @@ void planning_scene::PlanningScene::decoupleParent()
   parent_.reset();
 }
 
-void planning_scene::PlanningScene::setPlanningSceneDiffMsg(const moveit_msgs::PlanningScene &scene_msg)
+bool planning_scene::PlanningScene::setPlanningSceneDiffMsg(const moveit_msgs::PlanningScene &scene_msg)
 {
+  bool ret = true;
```

Comment:
```&=``` is a standard C operator, I don't see why we shouldn't use it.
```+=``` might even work, but how do you add boolean values?
Now *that* doesn't make sense to me :-)

The only alternatives I see to ```a&=b``` are ```a=a&b``` or ```a=a && b```.
If you favor one of them, I can change it. I don't really care...

btw, I just changed the variable name.